docs/mkdocs/docs/home/exceptions.md

```diff
@@ -900,3 +900,15 @@ A JSON Patch operation 'test' failed. The unsuccessful operation is also printed
     ```
     [json.exception.other_error.501] unsuccessful: {"op":"test","path":"/baz","value":"bar"}
     ```
+
+### json.exception.other_error.502
+
+when sax_parse operation has failed due to sax handler being nulllptr
```

Comment:
Please make a complete sentence out of this. Typo: "nulllptr" -> "null pointer".